packages/appstore/src/components/modals/mt5-migration-modal/mt5-migration-modal.tsx

```diff
@@ -0,0 +1,279 @@
+import React from 'react';
+import {
+    DesktopWrapper,
+    MobileDialog,
+    MobileWrapper,
+    Modal,
+    UILoader,
+    Text,
+    Icon,
+    Button,
+    Checkbox,
+    StaticUrl,
+    Div100vhContainer,
+} from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import { useMT5SVGEligibleToMigrate } from '@deriv/hooks';
+
+const MigrateAccount = observer(({ to, type }: { to: string; type: string }) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+    const getFromAccountIcon = () => {
+        switch (type) {
+            case 'derived':
+                return 'IcMt5SvgDerived';
+            case 'financial':
+                return 'IcMt5SvgFinancial';
+            default:
+                return '';
+        }
+    };
+
+    const getToAccountIcon = (to_type: string) => {
+        switch (to_type) {
+            case 'bvi_derived':
+                return 'IcMt5BviDerived';
+            case 'bvi_financial':
+                return 'IcMt5BviFinancial';
+            case 'vanuatu_derived':
+                return 'IcMt5VanuatuDerived';
+            case 'vanuatu_financial':
+                return 'IcMt5VanuatuFinancial';
+            default:
+                return '';
+        }
+    };
+
+    return (
+        <React.Fragment>
+            <Icon icon={getFromAccountIcon()} height={99} width={96} />
+            <Icon icon={is_mobile ? 'IcLongArrowDown' : 'IcLongArrowRight'} height={24} width={24} />
+            <Icon icon={getToAccountIcon(`${to}_${type}`)} height={99} width={96} />
+        </React.Fragment>
+    );
+});
+
+type TMT5MigrationModalProps = {
+    openPasswordModal: (acc_type: unknown) => void;
+};
+
+const MT5MigrationModal = observer(({ openPasswordModal }: TMT5MigrationModalProps) => {
+    const { ui } = useStore();
+    const {
+        svg_accounts_to_migrate,
+        no_of_svg_accounts_to_migrate,
+        eligible_account_to_migrate,
+        eligible_svg_to_bvi_derived_accounts,
+        eligible_svg_to_bvi_financial_accounts,
+        eligible_svg_to_vanuatu_derived_accounts,
+        eligible_svg_to_vanuatu_financial_accounts,
+    } = useMT5SVGEligibleToMigrate();
+
+    const { disableApp, enableApp, is_mt5_migration_modal_open, toggleMT5MigrationModal, is_mobile } = ui;
+    const [show_modal_front_side, setShowModalFrontSide] = React.useState(true);
+    const [is_checked, setIsChecked] = React.useState(false);
+    const modal_title = <Localize i18n_default_text='Enhancing your trading experience' />;
+    const header_size = is_mobile ? 'xs' : 's';
+    const title_size = is_mobile ? 'xxs' : 's';
+    const checkbox_text_size = is_mobile ? 'xxs' : 'xs';
+    const content_size = is_mobile ? 'xxxs' : 'xs';
+
+    const ShowFrontSide = () => (
```

Comment:
Can we move these components to individual files to reduce the side of this component and for maintainability?